src/PureCloud.Client/Apis/StationsApi.cs

```diff
@@ -21,23 +21,23 @@ public StationsApi(IHttpClientFactory httpClientFactory, IOptions<PureCloudJsonS
     }
 
     /// <inheritdoc />
-    public async Task DeleteStationAssociatedUserAsync(string stationId, CancellationToken cancellationToken = default)
+    public async Task<bool> DeleteStationAssociatedUserAsync(string stationId, CancellationToken cancellationToken = default)
     {
         ArgumentException.ThrowIfNullOrEmpty(stationId);
 
-        var uri = UriHelper.GetUri($"api/v2/stations/{stationId}/associateduser", null);
+        var uri = $"api/v2/stations/{Uri.EscapeDataString(stationId)}/associateduser";
```

Comment:
You don't need uri variable here. Use the uri directly in the request. Ally this feedback everywhere